main/src/mill/main/RunScript.scala

```diff
@@ -319,11 +320,45 @@ object RunScript {
       }
   }
 
+  def evaluateTasks1[T](
```

Comment:
I don't like the name, but I dont have enough context with regards to `evalutateTasks` to propose a better one

Reply:
What about `evaluateTasksNamed`?